packages/node_modules/@ciscospark/react-component-activity-text/src/index.js

```diff
@@ -38,19 +43,22 @@ export default function ActivityText(props) {
       h5: [],
       h6: []
     }, [], content);
-    const htmlContent = {__html: cleaned};
+    const linkifiedContent = linkifyHtml(cleaned, linkifyOptions);
```

Comment:
How does this work if the other clients return a `a` tag? Can linkify handle that or does it double wrap the link?